dangerfile.js

```diff
@@ -86,11 +86,15 @@ const noFlowFiles = newJsFiles
 
 raiseIssueAboutPaths(warn, noFlowFiles, '@flow');
 
+// based on assumptions from https://github.com/nodejs/node/wiki/ES6-Module-Detection-in-Node
+// basically it needs to contain at least one line with import or export at the
+// beginning of it, followed by a space; any content thereafter doesn't matter
+const esModuleRegex = /^(import|export)\s/g;
```

Comment:
It does, but it seems like it's using ES module-like syntax in a commonjs file? Does using flow automatically make a file an es module? If so, that's fine and we don't actually have to change anything.